src/core/file.cc

```diff
@@ -992,18 +1019,54 @@ xfs_concurrency_from_kernel_version() {
     return 0;
 }
 
+static const std::string NVME_STR = "nvme";
+static const std::string NG_STR = "ng";
+
+bool
+is_nvme_device(std::string name) {
+  size_t pos = name.find(NVME_STR);
```

Comment:
Checking if a block device is an NVME device by its block device filename doesn't look safe. Can major:minor pair be used for that? Or maybe there's a (Linux-specific) ioctl out there?

Reply:
We reflected your comment to use major:minor pair for checking if block device is NVMe or not. 
It will be included in the upcoming patch.